tests/hotplug/instancetype.go

```diff
@@ -182,4 +184,111 @@ var _ = Describe("[sig-compute]Instance Type and Preference Hotplug", decorators
 		Entry("with maxGuest and maxSockets defined", true),
 		Entry("without maxGuest and maxSockets defined", false),
 	)
+
+	DescribeTable("should reject live update when preference requirements are no longer met - bug #14595",
+		func(
+			createVM func(*virtv1.VirtualMachineInstance, string) (*virtv1.VirtualMachine, error),
+			updateVM func(*virtv1.VirtualMachine) (*virtv1.VirtualMachine, error),
+			errMatcher gomegatypes.GomegaMatcher,
+		) {
+			vmi := libvmifact.NewAlpine(
+				libnet.WithMasqueradeNetworking(),
+				libvmi.WithResourceMemory("1Gi"),
+			)
+			vmi.Namespace = testsuite.GetTestNamespace(vmi)
```

Comment:
There should be `libvmi.WithNamespace`

Reply:
Yeah I didn't realise that `GetTestNamespace(nil)` was valid here, I'll fold it in.